analyzer/codechecker_analyzer/cmd/check.py

```diff
@@ -183,6 +183,38 @@ def add_arguments_to_parser(parser):
                                     "overwrites only those files that were "
                                     "update by the current build command).")
 
+    parser.add_argument('--compile-uniqueing',
```

Comment:
Couldn't we use python [choices](https://docs.python.org/2.7/library/argparse.html#choices) here to define the exact values of this option (strict, alpha, etc.)?

Reply:
I think not because we accept a regex too which can be any valid python regex.